x/ibc-transfer/module.go

```diff
@@ -356,19 +352,10 @@ func (am AppModule) OnAcknowledgementPacket(
 			sdk.NewAttribute(types.AttributeKeyReceiver, data.Receiver),
 			sdk.NewAttribute(types.AttributeKeyDenom, data.Denom),
 			sdk.NewAttribute(types.AttributeKeyAmount, fmt.Sprintf("%d", data.Amount)),
-			sdk.NewAttribute(types.AttributeKeyAckSuccess, fmt.Sprintf("%t", ack.Success)),
+			sdk.NewAttribute(types.AttributeKeyAck, fmt.Sprintf("%v", ack)),
```

Comment:
yeah we should

Reply:
I will have it emit the result or error message as well

Reply:
instead of the bool

Reply:
updated, let me know if you agree, with specs as well

Reply:
I like the usage above:

`sdk.NewAttribute(types.AttributeKeyAckSuccess, fmt.Sprintf("%t", err != nil))`

We could use `%v` instead (or as well), but it would be nice to have them both the same (the attributes in the two different functions)

Reply:
ah yea, I agree. Will update in a followup. We discussed standardizing IBC events on the IBC call which I think would be very useful in cases like this where there are multiple way to represent the same information